AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -3531,12 +3643,58 @@ protected void showTagsDialog() {
         showDialogFragment(dialog);
     }
 
+    // init or reset api list
+    private void jsApiInit() {
+        mCardSuppliedApiVersion = "";
+        mCardSuppliedDeveloperContact = "";
+
+        for (int i = 0; i < sApiList.length; i++) {
+            mJsApiListMap.put(sApiList[i], false);
+        }
+    }
+
  /*
  Javascript Interface class for calling Java function from AnkiDroid WebView
 see card.js for available functions
  */
+    // list of api that can be accessed
+    private final String[] sApiList = {"toggleFlag", "markCard"};
```

Comment:
```suggestion
    private final String[] mApiList = {"toggleFlag", "markCard"};
```

This isn't static and though Hungarian Notation isn't great we are at consistent at the moment, should be member variable (and then all usage will need change to match)